app/models/vote.rb

```diff
@@ -0,0 +1,6 @@
+class Vote < ApplicationRecord
+  validates :user, uniqueness: { scope: :user_id,
+    message: "should happen once per user" }
```

Comment:
I think you misinterpreted the request we had. We wanted person to be able to vote for more than one work, but we didn't want users to be able to vote for _a single work_ more than once. 

Put another way, if I upvote DAMN by Kendrick Lamar, I can still upvote Titanic, but I can't vote for DAMN again.